### PR TITLE
chore: update generate key illustration to use a still image

### LINKED ISSUE #20243: Use static illustration on generating keys screens
We want to use static image instead of animation frames As mentioned [here](https://discord.com/channels/624634427930312714/1146065551060389943/1240984505356128268) on Discord we should only use a static illustration (no transition) on the Generating Keys screens

follow up to https://github.com/status-im/status-mobile/pull/20076

## Implementation

Update the illustration to the one quoted on discord

## Acceptance Criteria

Illustration is updated to use static image

### PR DESCRIPTION
**Pull request by jo-mut**
fixes https://github.com/status-im/status-mobile/issues/20243

### Summary
This pr updates the generate keys screen to use a still image

https://github.com/status-im/status-mobile/assets/28704507/bb131737-e9c9-4c60-901a-7b8360f4776a

status: ready

### CODE REVIEW

**Review by ilmotta — APPROVED** [review]

**Review by ilmotta — APPROVED** [review]

File: src/status_im/contexts/onboarding/generating_keys/view.cljs [hunk 1]
```diff
@@ -123,4 +103,4 @@
      (sequence-animation generate-keys-opacity saving-keys-opacity keys-saved-opacity)
```
Comment (J-Son89) [inline src/status_im/contexts/onboarding/generating_keys/view.cljs]: we should also be deleting these opacities no? they're unused now.
There was also some mechanism @briansztamfater had used to check the status ok keys generated to block the transition until it was ready. Might be good to revisit "when" the keys are generated in the onboarding flow so this page transitions faster too. I discussed this before with @cammellos, basic idea is to actually do the background generation once we get the minimum amount of required data in the onboarding flow so this page is just for "show" and not indicative of what is actually happening at that time.

**Comment by qoqobolo** [comment]
Hey @jo-mut , thanks for the PR!
LGTM, but I have one question regarding this screen:
what do you think about adjusting the position of the illustration on the screen in this PR?
You can see the comparison of how the illustration is placed on different screens on real devices _vs._ how it looks in designs.
Would you like to fix it now (if it's possible and an easy fix) or should we (probably) log it separately? 


[Design:](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=12860-105360&t=HyTaEQLfBQFQq4g5-4)

Screenshot 2024-06-20 at 16 58 11

Pixel 7a:

Screenshot 2024-06-20 at 16 54 56

iPhone SE:

Screenshot 2024-06-20 at 16 55 31

iPhone 12 mini:

Screenshot 2024-06-20 at 16 55 58

Huawei P40 lite:

Screenshot 2024-06-20 at 16 56 18

**Comment by ilmotta** [comment]
> Would you like to fix it now (if it's possible and an easy fix) or should we (probably) log it separately?

@qoqobolo, I think this problem can be relatively tricky to get right because it has to be well tested in different devices with different screen resolutions and aspect ratios. If we could do in a separate issue it would be nice. In my opinion it's an important & old bug you are mentioning and makes our on-boarding flows look quite unprofessional in all my Android devices.

**Comment by jo-mut** [comment]
I agree with @ilmotta . Lets merge this and look into the issue separately

**Comment by qoqobolo** [comment]
Thanks for the answer @ilmotta! I'll log a separate issue for this. 

> I agree with @ilmotta . Lets merge this and look into the issue separately

@jo-mut please take a look at Jamie's [comment](https://github.com/status-im/status-mobile/pull/20506#discussion_r1647584356) and let me know if PR is ready to be merged or needs some changes.
Thank you!

File: src/status_im/contexts/onboarding/generating_keys/view.cljs (hunk 1 above)
Comment (jo-mut) [reply src/status_im/contexts/onboarding/generating_keys/view.cljs]: @J-Son89 these are still being used by the screen title. I only updated the illustration for this reason unless we also just want to show a static title

Comment (J-Son89) [reply src/status_im/contexts/onboarding/generating_keys/view.cljs]: Thanks!

Comment (jo-mut) [reply src/status_im/contexts/onboarding/generating_keys/view.cljs]: I did not remove them instead just refactored the code to have them in the title component which is the only place they are being used.

**Comment by jo-mut** [comment]
@qoqobolo the pr is okay for merge. Thank you

**Comment by qoqobolo** [comment]
Thank you @jo-mut!

**Pull request merged**